cardano_node_tests/utils/tx_view.py

```diff
@@ -243,4 +263,10 @@ def check_tx_view(  # noqa: C901
     ):
         raise AssertionError("collateral inputs are not the expected")
 
+    # check reference inputs, this is only available on Alonzo+ TX
+    if loaded_tx_version >= VERSIONS.ALONZO and not _check_reference_inputs(
```

Comment:
Please use named args for `_check_reference_inputs` (in general always use named args when passing more than one argument).